specs/icmp.rflx

```diff
@@ -0,0 +1,27 @@
+package ICMP is
+
+    type Tag is (Echo_Reply => 0, Echo_Request => 8)
+        with Size => 8;
```

Comment:
The indentation should be 3 spaces (like in Ada).

Reply:
fixed and squashed into 6d14bb9

Reply:
It is not fixed.

Reply:
now it is, i thought you only meant with clauses